website/src/scss/sections/project.scss

```diff
@@ -0,0 +1,100 @@
+@import '../parts/utils/fonts';
+@import '../parts/utils/colors';
+
+.project {
+  padding-top: 90px;
+  padding-bottom: 100px;
+  padding-left: 32px;
+  padding-right: 32px;
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+
+  &__content {
+    flex: 0 0 100%;
+    width: 100%;
+    padding-left: 220px;
+    padding-bottom: 75px;
+  }
+
+  &__title {
+    font-size: 64px;
+    line-height: 1.03;
+    letter-spacing: -2px;
```

Comment:
Typography styles should somehow be separated / extracted in a map, ideally when you have a consistent design you'd have them all in 1 place.